tracee-rules/signatures/rego/dynamic_code_loading.rego

```diff
@@ -0,0 +1,29 @@
+package main
+
+import data.tracee.helpers
+
+__rego_metadoc__ := {
+    "name": "Dynamic Code Loading",
+    "description": "writing to executable allocated memory region",
+    "tags": ["linux", "container"],
+    "properties": {
+        "Severity": 2,
+        "MITRE ATT&CK": "Defense Evasion: Obfuscated Files or Information",
+    }
+}
+
+eventSelectors := [
+    {
+        "source": "tracee",
+        "name": "mem_prot_alert"
+    }
+]
+
+tracee_selected_events[eventSelector] {
+	eventSelector := eventSelectors[_]
+}
+
+
+tracee_match {
+    input.eventName == "mem_prot_alert"
```

Comment:
Can you please set this in
https://github.com/aquasecurity/tracee/blob/main/entrypoint.sh#L38

Reply:
I think this can be a different PR (assigned to someone else, if Roi prefers), we can add signatures to this library even if the default entrypoint doesn't detect them, just like execenv isn't enabled.
I'm suggesting this just to speed things up, Roi if you want to include it go for it

Reply:
so after talking about it we decided to do it in a different PR.
resolving this conversation.